zulipterminal/core.py

```diff
@@ -262,6 +263,17 @@ def _finalize_show(self, w_list: List[Any]) -> None:
         if focus_position >= 0 and focus_position < len(w_list):
             self.model.msg_list.set_focus(focus_position)
 
+    def deregister_client(self) -> None:
+        try:
+            queue_id = self.model.queue_id
+            self.client.deregister(queue_id)
+        except Exception as e:
```

Comment:
@realsdx we don't need to a try-except statement here since we are already handling general exception already in `run.py`. you can simply deregister from the queue here.

Reply:
Okay.